src/jit/emitxarch.cpp

```diff
@@ -7135,7 +7135,7 @@ void emitter::emitIns_Call(EmitCallType          callType,
                         // An absolute indir address that doesn't need reloc should fit within 32-bits
                         // to be encoded as offset relative to zero.  This addr mode requires an extra
                         // SIB byte
-                        noway_assert(static_cast<int>(reinterpret_cast<intptr_t>(addr)) == (size_t)addr);
+                        noway_assert(static_cast<size_t>(reinterpret_cast<intptr_t>(addr)) == (size_t)addr);
```

Comment:
This change is incorrect. The assert will never fire with your change. Please read the comment above.

Reply:
let me drop this change. I want to fix the blocking gcc compilation issues first. This is a nice to have (nuisance)